libs/common/src/services/api.service.ts

```diff
@@ -461,6 +462,10 @@ export class ApiService implements ApiServiceAbstraction {
     return this.send("PUT", "/accounts/update-temp-password", request, true, false);
   }
 
+  putUpdateTdeOffboardingPassword(request: UpdateTdeOffboardingPasswordRequest): Promise<any> {
+    return this.send("PUT", "/accounts/update-tde-offboarding-password", request, true, false);
+  }
```

Comment:
Do you think it makes sense to put this in `DevicesApiService`? I know its more TDE related than devices, but I think the `DeviceTrustService` should be responsible for building the request and calling the api service.

Also:
```suggestion
  putUpdateTdeOffboardingPassword(request: UpdateTdeOffboardingPasswordRequest): Promise<void> {
    return this.send("PUT", "/accounts/update-tde-offboarding-password", request, true, false);
  }
```

Reply:
`DeviceTrustService` feels more like it's about managing devices, and their trust, themselves (and looking into it, it only is concerned with that) rather than being about functionality around the TDE feature. Maybe a refactor into "TrustedDeviceEncryptionFeatureService" or something along the lines would make sense. I'll keep it in apiservice for now.